src/electionguard_gui/services/election_service.py

```diff
@@ -45,6 +46,7 @@ def create_election(
             "key_ceremony_id": key_ceremony.id,
             "guardians": context.number_of_guardians,
             "quorum": context.quorum,
+            "election_url": election_url,
```

Comment:
Forgot to save the url in a prior PR.  It was being put into the context, but now it gets displayed in the UI too, which is kinda nice.